src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentForm.tag

```diff
@@ -44,8 +44,8 @@
             <tbody>
                 <tr>
                     <th class="text-center">User/Group</th>
-                    <th class="text-center">Can see your comment</th>
-                    <th class="text-center">Can see your name</th>
+                    <th class="text-center">Can see this comment</th>
+                    <th class="text-center">Can see giver's name</th>
```

Comment:
Might be possible, switching it to comment giver would be good then, I'll extend this to student comments as well to preserve consistency.